INSTALL.md

```diff
@@ -5,6 +5,8 @@ System requirements: GalSim currently only supports Linux and Mac OSX.
 
 Table of Contents:
 
+0. [Overall summary](#0-overall-summary)
```

Comment:
Update.  Apparently Github did finally fix it for comments.  (I used `0.` for each of the above list items.)  But apparently not for the file markdown parsing (based on Josh's view of this file that he posted).